explorer/src/data/countriesData.ts

```diff
@@ -0,0 +1,39 @@
+export const countriesData = [
```

Comment:
You can fetch this from the explorer API - https://testnet-milhon-explorer.nymtech.net/api/countries